plugins/ConnectedLivingSpace/ModuleDockingHatch.cs

```diff
@@ -64,7 +96,7 @@ private void OpenHatch()
         }
 
         [KSPEvent(active = true, guiActive = true, guiName = "Close Hatch")]
-        private void CloseHatch()
+        public void CloseHatch()

```

Comment:
Originally I was going to expose this method, but determined that a change to HatchOpen would work.   Can stay private.